src/System.Diagnostics.Process/src/System/Diagnostics/Process.Linux.cs

```diff
@@ -64,37 +64,41 @@ internal DateTime StartTimeCore
         /// <returns>The converted time.</returns>
         internal static DateTime BootTimeToDateTime(TimeSpan timespanAfterBoot)
         {
-            // Use the uptime and the current time to determine the absolute boot time.
-            DateTime bootTime = DateTime.UtcNow - Uptime;
-
             // And use that to determine the absolute time for timespan.
-            DateTime dt = bootTime + timespanAfterBoot;
+            DateTime dt = BootTime + timespanAfterBoot;
 
             // The return value is expected to be in the local time zone.
             // It is converted here (rather than starting with DateTime.Now) to avoid DST issues.
             return dt.ToLocalTime();
         }
 
-        /// <summary>Gets the elapsed time since the system was booted.</summary>
-        private static TimeSpan Uptime
+        /// <summary>Gets the system boot time.</summary>
+        private static DateTime GetBootTime()
         {
-            get
+            // '/proc/stat -> btime' gets the boot time.
```

Comment:
Can you add a note:
'btime is the time of system boot in seconds since the Unix epoch. It includes suspended time and is updated based on the system time (settimeofday).'

Reply:
Done